docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -59,10 +64,16 @@ def transform_fn(model, request_body, input_content_type, output_content_type="a
 
     pred = model.predict(data)
     pred_proba = model.predict_proba(data)
-    prediction = pd.concat([pred, pred_proba], axis=1).values
+    prediction = pd.concat([pred, pred_proba], axis=1)
 
-    return json.dumps(prediction.tolist()), output_content_type
+    return prediction.to_json(), output_content_type
 ```
+For inference with other types of AutoGluon Predictors, i.e. TextPredictor, the inference script you provided will be quite similar to the one above.
+Mostly, you just need to replace `TabularPredictor` to be `TextPredictor` for example.
+
+### Note on image modality
+To do inference on image modality, you would need to embed the image info, as bytes for example, into a column of the test data.
+Then in the inference container, you would need to store the image into the disk and update the test data with the image paths accordingly.
```

Comment:
This description is kind of too high-level. Could we add some code examples?

Reply:
Good suggestion!